index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/73a06aba35.js" crossorigin="anonymous"></script>
+    <title>Portfolio2</title>
+  </head>
+  <body>
+
+
+    <div class="container">
+      <header>
```

Comment:
- [ ]  the navbar stylings are not matching Figma when it comes to margins and padding kindly try to fix it : 


**_- your design :_**

![image](https://user-images.githubusercontent.com/89282221/185762387-1ec49339-80c3-4245-b3a3-5982971de5b7.png)

**_- Figma design :_**
 
![image](https://user-images.githubusercontent.com/89282221/185762446-d8010b20-c746-4cde-bca3-e0f96e3c1088.png)

Reply:
done